### DISCUSSION TITLE
ERR! Exit status 126

### DISCUSSION THREAD

**Discussion by Raphael-ui** [vercel/next.js#19393] [Help]
15:40:49	Cloning github.com/Raphael-ui/Components (Branch: master, Commit: f12ce9a)
15:41:00	Cloning completed in 11673ms
15:41:00	Analyzing source code...
15:41:15	Installing build runtime...
15:41:15	Build runtime installed: 511.056ms
15:41:16	Looking up build cache...
15:41:16	Build cache not found
15:41:17	Installing dependencies...
15:41:28	npm WARN @vue/compiler-sfc@3.0.2 requires a peer of vue@3.0.2 but none is installed. You must install peer dependencies yourself.
15:41:28	npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/watchpack/node_modules/fsevents):
15:41:28	npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
15:41:28	npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fsevents):
15:41:28	npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
15:41:28	up to date in 10.587s
15:41:29	78 packages are looking for funding
15:41:29	  run `npm fund` for details
15:41:29	Running "npm run build"
15:41:30	> card-component@0.1.0 build /vercel/workpath0
15:41:30	> vue-cli-service bsuduild
15:41:30	sh: /vercel/workpath0/node_modules/.bin/vue-cli-service: Permission denied
15:41:30	npm ERR! code ELIFECYCLE
15:41:30	npm ERR! errno 126
15:41:30	npm ERR! card-component@0.1.0 build: `vue-cli-service bsuduild`
15:41:30	npm ERR! Exit status 126
15:41:30	npm ERR! 
15:41:30	npm ERR! Failed at the card-component@0.1.0 build script.
15:41:30	npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
15:41:30	npm ERR! A complete log of this run can be found in:
15:41:30	npm ERR!     /vercel/.npm/_logs/2020-11-21T14_41_30_265Z-debug.log
15:41:30	Error: Command "npm run build" exited with 126
15:41:34	Done with "package.json"

**Comment by crecordz** [comment]
I have the same error. Did you fix it?

**Reply by Jacqueline-Kelly** [reply]
I also have the same error! Any notes?
